Resources/Prototypes/Datasets/tips.yml

```diff
@@ -136,3 +136,12 @@
   - "Lottery crates can very rarely contain The Throngler."
   - "You can tell if an area with firelocks up is spaced by looking to see if the firelocks have lights beside them."
   - "Instead of picking it up, you can alt-click food to eat it. This also works for mice and other creatures without hands."
+  # CD tips:
+  - "Upon entering crysleep your items will be transferred to the lost and found locker in the HoP's office"
+  - "It is possible to return to arrivals using the arrivals shuttle. This can be used for easy access to cryosleep pods"
+  - "Everyone is permitted to commit minor crimes. You can always ahelp if you want to commit a larger crime!"
```

Comment:
Probably fine, but with how many issues we've had in the past with crimes, maybe change it to "it's advised to ahelp before committing crimes" or something along those lines.